<launch>
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="RatSLAMLocalViewCells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_stlucia.txt.in" />
        <param name="use_sim_time" value="true" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_pc"
        name="RatSLAMPoseCells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_stlucia.txt.in" />
        <param name="use_sim_time" value="true" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_em"
        name="RatSLAMExperienceMap"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_stlucia.txt.in" />
        <param name="use_sim_time" value="true" />
    </node>

	<node
        pkg="ratslam"
        exec="ratslam_vo"
        name="RatSLAMVisualOdometry"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_stlucia.txt.in" />
        <param name="use_sim_time" value="true" />
    </node>
</launch>

 
